src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -837,6 +842,10 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
         cmd_helm_upgrade.extend(["--set", "global.httpProxy={}".format(http_proxy)])
     if no_proxy:
         cmd_helm_upgrade.extend(["--set", "global.noProxy={}".format(no_proxy)])
+    if https_proxy or http_proxy or no_proxy:
+        cmd_helm_upgrade.extend(["--set", "global.isProxyEnabled={}".format(True)])
+    if disable_proxy:
+        cmd_helm_upgrade.extend(["--set", "global.isProxyEnabled={}".format(False)])
```

Comment:
I think we should handle the case where customer by mistake pass proxy params as well as --disable proxy.

Reply:
done